nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/java/org/nuxeo/ecm/platform/comment/TestCommentsMigrator.java

```diff
@@ -241,33 +206,29 @@ public void testProbe() {
 
         CommentServiceConfig config = newConfig();
         CommentManager relationCommentManager = new CommentManagerImpl(config);
-        CommentManager propertyCommentManager = new PropertyCommentManager();
 
         DocumentModel domain = session.createDocumentModel("/", "test-domain", "Domain");
         session.createDocument(domain);
-        DocumentModel file = session.createDocumentModel("/test-domain", "file1", "File");
+        DocumentModel file = session.createDocumentModel("/test-domain", "anotherFile", "File");
         file = session.createDocument(file);
         session.save();
 
-        // no comments, detected as already migrated
         Migrator migrator = new CommentsMigrator();
+        assertEquals(MIGRATION_STATE_RELATION, migrator.probeState());
+
+        // Migrate the created relation based comment to property
+        migrateFromRelationToProperty(migrator);
+
+        // No more relation comments detected as 'Relation'
         assertEquals(MIGRATION_STATE_PROPERTY, migrator.probeState());
 
-        // just a relation-based comment, detected as not migrated
+        // Just a relation-based comment, detected as not migrated
         DocumentModel comment = session.createDocumentModel(null, "comment", COMMENT_DOC_TYPE);
         comment.setPropertyValue(COMMENT_AUTHOR, session.getPrincipal().getName());
         comment = relationCommentManager.createComment(file, comment);
         session.save();
         assertEquals(MIGRATION_STATE_RELATION, migrator.probeState());
 
-        // both a relation-based comment and a property-based comment, detected as not migrated
-        DocumentModel otherComment = session.createDocumentModel(null, "comment", COMMENT_DOC_TYPE);
```

Comment:
We should keep this block.

Reply:
please see https://github.com/nuxeo/nuxeo/pull/3289#discussion_r318125639, now there no need to create a new comment, as they are created by the `@before`

Reply:
Ok for document creation, but the assertion of `probeState` for a repository containing both relation and property comments was lost (at least we don't test it explicitly).